internal/types/sidecarconfig.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 SAP SE or an SAP affiliate company
```

Comment:
yes that would make more sense.

Reply:
Thinking again, the file name should just be called config.go and the struct should be called BackupRestoreConfig instead of the current SidecarConfig. WDYT?